data/911/index.html

```diff
@@ -17,33 +17,37 @@
   downloads:
     - url: https://drive.google.com/a/utah.gov/uc?id=1Sp6v4FmNV5fhdfEWPTxPGHMbnz-yeY47&export=download
       label: 'UtahNG911GIS: File Geodatabase'
-  updates:
-    - May 20, 2020
-    - April 08, 2020
-    - February 28, 2020
-    - February 7, 2020
-    - January 2, 2020
-    - October 9, 2019
-    - September 5, 2019
-    - August 8, 2019
-    - July 10, 2019
-    - June 5, 2019
-    - May 8, 2019
-    - April 2, 2019
-    - February 23, 2019
-    - January 24, 2019
-    - November 29, 2018
-    - October 11, 2018
-    - August 14, 2018
-    - June 27, 2018
-    - May 30, 2018
-    - April 12, 2018
-    - March 16, 2018
-    - February 14, 2018
-    - February 6, 2018
-    - January 19, 2018
-    - January 04, 2018
-    - December 18, 2017
+  updated:
+    - June 24, 2020
+    <details>
+      <summary>Previous Updates</summary>
+      - May 20, 2020
+      - April 08, 2020
+      - February 28, 2020
+      - February 7, 2020
+      - January 2, 2020
+      - October 9, 2019
+      - September 5, 2019
+      - August 8, 2019
+      - July 10, 2019
+      - June 5, 2019
+      - May 8, 2019
+      - April 2, 2019
+      - February 23, 2019
+      - January 24, 2019
+      - November 29, 2018
+      - October 11, 2018
+      - August 14, 2018
+      - June 27, 2018
+      - May 30, 2018
+      - April 12, 2018
+      - March 16, 2018
+      - February 14, 2018
+      - February 6, 2018
+      - January 19, 2018
+      - January 04, 2018
+      - December 18, 2017
```

Comment:
**thought**: i do have a hard time think of reasons to how these old dates provide value

Reply:
see comment above.  but, it looks terrible on the data page.  i might just remove them and put a note in the data description.  i think I'll go that route.